### PR TITLE
Adding a new editorial for the problem LOJ-1509

### PR DESCRIPTION
**Pull request by nfpranta**

### CODE REVIEW

File: 1509/en.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+### Approach:
+This is a very simple adhoc problem.The current team will only not be able to become the champion of the contest if they have solved all the problems that are in the problem statement and still couldn't achieve the first position.Other than this there is always a chance for the current team to become champion after the end of the contest.
+
+If you are still stuck with this problem, check the code below:
+
+### C++
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+#define int long long int
```
Comment (rebornplusplus) [inline 1509/en.md:16]: Let's not define `int` to be `long long int`.

File: 1509/en.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    {
+        int p, s, r;
+        cin >> p >> s >> r;
+        cout << "Case " << k << ": ";
+        if (p == s and r != 1)
+        {
+            cout << "No" << endl;
+        }
+        else
+            cout << "Yes" << endl;
```
Comment (rebornplusplus) [inline 1509/en.md:31]: Shouldn't this be wrapped in braces as well, by convention?

**Review by rebornplusplus — CHANGES_REQUESTED** [review]
Looks good.

I only made a few minor comments that need to be addressed.

**Comment by nfpranta** [comment]
> Looks good.
> 
> I only made a few minor comments that need to be addressed.

**Comment by nfpranta** [comment]
I think I have made the changes you asked for.Is it ok now?

**Review by rebornplusplus — APPROVED** [review]
Looks good now. Thanks!

**Pull request merged**